src/index.html

```diff
@@ -18,5 +18,66 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="aimg"
+        href="dfrf"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'logo', a more descriptive text would be better, like 'Moyo Logo'.